charts/tidb-lightning/values.yaml

```diff
@@ -91,9 +91,9 @@ annotations: {}
 tolerations: []
 affinity: {}
 
-# The delivery backend used to import data (valid options include `importer`, `local` and `tidb`).
+# The delivery backend used to import data (valid options include `local`, `tidb` and `importer`(deprecated)).
 # If set to `local`, then the following `sortedKV` should be set.
-backend: importer
+backend: local
 
 # For `local` backend, an extra PV is needed for local KV sorting.
 sortedKV: {}
```

Comment:
The `storageClassName` and `storage` should be uncommented.

Reply:
Done